src/main/java/com/faforever/api/data/domain/Vote.java

```diff
@@ -21,6 +22,7 @@
 @Include(type = Vote.TYPE_NAME, rootLevel = true)
 @ReadPermission(expression = IsEntityOwner.EXPRESSION)
 @UpdatePermission(expression = "Prefab.Role.None")
+@EqualsAndHashCode(of = {"player", "votingSubject"}, callSuper = false)
```

Comment:
This could be removed if comment in AbstractEntity is considered. The id alone should be enough.

Reply:
Ok. Accepted :)